SWLOR.Game.Server/ValueObject/EffectiveStats.cs

```diff
@@ -18,6 +18,7 @@ public class EffectiveItemStats
         public int Fabrication { get; set; }
         public int Armorsmith { get; set; }
         public int Harvesting { get; set; }
+        public int Scavenging { get; set; }
```

Comment:
You need to add a line to PlayerStatService.GetPlayerItemEffectiveStats to calculate this new property. Otherwise it'll be 0 every time.

Reply:
Done - and made some other related changes to make this work end to end.